extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -115,17 +119,68 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
       (event) => event.stopPropagation(),
       true
     );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (currPageId) => {
+        if (currPageId != this.getPageEl().getAttribute('id')) {
+          return;
+        }
+        const DURATION_MS = 1500;
+        let startTime;
+        const animateFrame = (currTime) => {
+          // Set current startTime if not defined.
+          if (!startTime) {
+            startTime = currTime;
+          }
+          const elapsed = currTime - startTime;
+          if (elapsed < DURATION_MS) {
```

Comment:
We can invert this `if` and have an early return: `if (elapsed > DURATION_MS) {return;}`